gradle.properties

```diff
@@ -1,26 +1,9 @@
 sovityEdcExtensionsVersion=0.0.1-SNAPSHOT
 sovityEdcExtensionGroup=de.sovity.edc.ext
+sovityBrokerServerGroup=de.sovity.broker
 sovityEdcGroup=de.sovity.edc
 edcGroup=org.eclipse.edc
```

Comment:
same with the edc dependencies

Reply:
ok, cleared

I also found a lot of deps that I didn't switch to the new format